_internals/tds-protocol.md

```diff
@@ -6,84 +6,39 @@ nav_order: 8
 
 ## The TDS wire protocol
 
-Babelfish exposes PostgreSQL as an Microsoft SQL Server. To achieve that, the community has
-implemented the TDS wire protocol which forms the backbone of the entire solution.
-In this chapter, we want to take a deeper look at the details of the protocol's
-inner workings, and understand how things happen.
+Babelfish implements SQL Server behavior over the TDS wire protocol on a PostgreSQL server.  Babelfish supports connection over the following networking types with supporting protocols: 
 
-### Basic message flow
-
-Let's first dive into the most basic flow of messages.
-The following [image](https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-tds/ms-tds_files/image001.png) shows the
-basic communication exchange on a high level.
-
-As a database protocol, TDS relies on network transmission order and expects a
-reliable connection which basically means ...
-
-- TCP
+- [TCP](https://en.wikipedia.org/wiki/Transmission_Control_Protocol)
 - VIA (Virtual Interface Architecture)
-- Named pipes
+- [Named pipes](https://en.wikipedia.org/wiki/Named_pipe)
```

Comment:
Named pipes are NOT supported (I checked with Kuntal)

Reply:
Removed... thank you!